src/lib.rs

```diff
@@ -548,9 +513,20 @@ decl_storage! {
         /// Post identifier value to be used for for next post created.
         pub NextPostId get(next_post_id) config(): PostId;
 
+        /// Max depth of category.
+        pub MaxCategoryDepth get(max_category_depth) config(): u8;
+
         /// Account of forum sudo.
         pub ForumSudo get(forum_sudo) config(): Option<T::AccountId>;
 
+        /// Moderator set for each Category
+        pub CategoryByModerator get(category_by_moderator) config(): double_map CategoryId, blake2_256(T::AccountId) => bool;
+
+        pub ReactionByPost get(reaction_by_post) config(): double_map PostId, blake2_256(T::AccountId) => PostReaction;
+
+        pub PollDesc get(poll_desc) config(): double_map ThreadId, blake2_256(u8) => Vec<u8>;
```

Comment:
What is this field? I assume its a human readable description text? If so, why does it not live in `Thread` ?

See comment about `Thread::include_poll`.

Reply:
Yes, these descriptions are text for human reading. At first, I put the poll it in the thread. Later, I consider scenarios about changes. For example, thread created without poll, then add one. Or thread with poll, but then delete it for some reason.

Reply:
Storage size is also a factor, most of cases, we just need some basic information from thread. It can avoid load too much data from db.